ex12_ch3.py

```diff
@@ -0,0 +1,25 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+"""
+Created on Thu May 10 15:34:22 2018
+
+@author: amyhaddad
+"""
+
+#step1: get input for principal amount
+principal_amount = int(input("Enter the principal amount: $"))
+
+#step2: get input for interest rate as a percent (ie, "15" and not ".15")
+interest_rate = float(input("Enter the interest rate: "))
+
+#step3: turn interest rate into decimal
+interest_rate_as_decimal = (interest_rate/100) 
```

Comment:
I'd put spaces around the division and the parentheses aren't strictly needed, although they have no effect:

```
interest_rate_as_decimal = interest_rate / 100
```